### ISSUE TITLE
Large difference between my result of s3dis and yours

### ISSUE THREAD

**Issue by yzgfjly** [yangyanli/PointCNN#52]
I tried to train and test S3DIS on TITAN X of 12GB memory. And because of OOM, I changed the batchsize from 16 to 12.
The model is trained on Area2~6, and tester on Area1.
The result is listed as follows:

> Overall accuracy: 0.6220546181741131
IoU:
0.4692968354568871
0.6634375532984669
0.6585915255348683
0.6286324167941533
0.4089729327212604
0.4101586582534994
0.6679818077667772
0.5368703770020128
0.4935388803232025
0.47200222643714557
0.3127629041882209
0.38665093766294856
0.027841126074583687
mean IoU:
0.4720567831933866

The mean IoU of my result is 47.21%, which is much less than your result: segmentation mean IoU on S3DIS (62.74%).
I am confused that whether the difference between batchsize could bring such large difference.

Also, is it possible that I can train the model on two GPUs at the same time? In this way, I may set the batchsize as 16 and evaluate again.

**Comment by yzgfjly** [comment]
Thanks for your reply!
And you mentioned that the result on S3DIS(62.74%) was tested on 6 cross areas. Do you mean that I should train model on Area2~6, and test on Area1~6, or train the model on Area2~6, test on Area1, then train another model on Area1,3,4,5,6, test on Area2, then train another model on Area1,2,4,5,6, test on Area3, and so on, then calculate the average result of these 6 experiments？
Besides, I used the latest code and config file.
Thanks!

**Comment by burui11087** [comment]
@yzgfjly 
The second one is correct. I think you can get same result reported in paper if you choose per class segmentation  accuracy highest model.
Thanks

**Comment by meteorshowers** [comment]
@burui11087  do you use pretrained model to get 65% performance? or something others?

**Comment by burui11087** [comment]
@meteorshowers We get 65% performance by training model directly, meaning that nothing used.
